backend-example/src/browser/command-menu-contribution.ts

```diff
@@ -1,49 +1,31 @@
-import { Command, CommandContribution, CommandRegistry, MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from '@theia/core/lib/common';
+import { Command, CommandContribution, CommandRegistry} from '@theia/core/lib/common';
 import { inject, injectable } from 'inversify';
-import { BackendWithClient, BackendWithoutClient } from '../common/protocol';
+import { HelloBackendWithClientService, HelloBackendService } from '../common/protocol';
 
-const MY_MAIN_MENU = [...MAIN_MENU_BAR, '9_mymenu'];
-
-const WithClientCommand: Command = {
-    id: 'with-client.command',
-    label: 'Greet From Backend With Client',
+const SayHelloViaBackendCommandWithCallBack: Command = {
+    id: 'sayHelloOnBackendWithCallBack.command',
+    label: 'Say hello on the backend with a callback to the client',
 };
 
-const WithoutClientCommand: Command = {
-    id: 'without-client.command',
-    label: 'Greet From Backend Without Client',
+const SayHelloViaBackendCommand: Command = {
+    id: 'sayHelloOnBackend.command',
+    label: 'Say hello on the backend',
 };
 
 @injectable()
 export class BackendExampleCommandContribution implements CommandContribution {
 
     constructor(
-
-        @inject(BackendWithClient) private readonly backendWithClient: BackendWithClient,
-        @inject(BackendWithoutClient) private readonly backendWithoutClient: BackendWithoutClient,
+        @inject(HelloBackendWithClientService) private readonly backendWithClient: HelloBackendWithClientService,
```

Comment:
I think it would be more clear if we unify the names, eg:
`backendWithClient` and `backendService` or 
`helloBackendWithClient` and `helloBackendService`

Reply:
done